tests/schemas/invalid/test_invalid_text_field_suggestions.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Is this schema needed?. We don't normally commit schemas to test raw JSON schema validation. As you are just testing the "required" rule I don't think we need this.

Reply:
Schema removed, thanks!